portality/app.py

```diff
@@ -49,22 +49,22 @@
 app.register_blueprint(account, url_prefix='/account') #~~->Account:Blueprint~~
 app.register_blueprint(admin, url_prefix='/admin') #~~-> Admin:Blueprint~~
 app.register_blueprint(publisher, url_prefix='/publisher') #~~-> Publisher:Blueprint~~
-app.register_blueprint(query, url_prefix='/query') # ~~-> Query:Blueprint~~
-app.register_blueprint(query, url_prefix="/admin_query")
-app.register_blueprint(query, url_prefix="/publisher_query")
-app.register_blueprint(query, url_prefix="/editor_query")
-app.register_blueprint(query, url_prefix="/associate_query")
+app.register_blueprint(query,  name='query', url_prefix='/query') # ~~-> Query:Blueprint~~
+app.register_blueprint(query,  name='admin_query', url_prefix='/admin_query')
+app.register_blueprint(query,  name='publisher_query', url_prefix='/publisher_query')
+app.register_blueprint(query,  name='editor_query', url_prefix='/editor_query')
+app.register_blueprint(query,  name='associate_query', url_prefix='/associate_query')
```

Comment:
In flask 2.0,  exception will be raise if duplicate register_blueprint 
attribute `name=` can solve exception